### ISSUE TITLE
requirements.txt should not be deployed

### ISSUE THREAD

**Issue by wat-usmdon** [influxdata/influxdb-client-python#229]
__Steps to reproduce:__
List the minimal actions needed to reproduce the behavior.

1. Install influxdb_client. pip install influxdb_client.

```
# dnf install -y python3-pip
# pip3 install influxdb_client
```

__Expected behavior:__
Only the influxdb_client library files should be deployed.

__Actual behavior:__
The influxdb_client requirements.txt file gets deployed to ~/.local/requirements.txt or /usr/local/requirements.txt.

```
# cat /usr/local/requirements.txt
rx >= 3.0.1
certifi >= 14.05.14
six >= 1.10
python_dateutil >= 2.5.3
setuptools >= 21.0.0
urllib3 >= 1.15.1
pytz>=2019.1
```

Additionally, the files ciso-requirements.txt extra-requirements.txt and test-requirements.txt are put at the same location.

__Specifications:__
 - Client Version: 1.11.0, 1.16.0
 - InfluxDB Version: N/A
 - Platform: Ubuntu Focal, Centos 8

__Other:__

I found this because of another bug openshift/openshift-restclient-python#403. The combination of these two bugs means you cannot successfully run `pip3 install influxdb_client openshift` because one tries to create the path as a directory and the other tries to create the path as a file.

```
# pip3 install influxdb_client openshift
...
    error: could not create '/usr/local/requirements.txt': File exists

    ----------------------------------------
Command "/usr/bin/python3.6 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-har3wcw1/openshift/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-ttrrwxh5-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-build-har3wcw1/openshift/
```

```
# pip3 install openshift influxdb_client
...
IsADirectoryError: [Errno 21] Is a directory: '/usr/local/requirements.txt'
```

**Comment by bednar** [comment]
@wat-usmdon, thanks for using our client, We will take a look.

**Comment by bednar** [comment]
Hi @wat-usmdon,

Could you check that your issue is fixed by #232? You could install the fixed version by

```
pip install git+https://github.com/influxdata/influxdb-client-python.git@chore/specify-requirements-in-setup.py
```

Regards

**Comment by wat-usmdon** [comment]
That works for me. Thank you.
